README.md

```diff
@@ -1,6 +1,6 @@
 # :mag_right: ​AnaL&oacute;g
 
-[![Travis (.com)](https://img.shields.io/travis/com/toparvion/analog?style=plastic)](https://travis-ci.com/github/Toparvion/analog) [![Sonar Coverage](https://img.shields.io/sonar/coverage/Toparvion_analog?server=https%3A%2F%2Fsonarcloud.io&style=plastic)](https://sonarcloud.io/dashboard?id=Toparvion_analog) [![Sonar Quality Gate](https://img.shields.io/sonar/quality_gate/Toparvion_analog?server=https%3A%2F%2Fsonarcloud.io&style=plastic)](https://sonarcloud.io/dashboard?id=Toparvion_analog) [![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/toparvion/analog?color=green&sort=semver&style=plastic)](https://github.com/Toparvion/analog/releases/latest) [![GitHub](https://img.shields.io/github/license/toparvion/analog?style=plastic)](https://github.com/Toparvion/analog/blob/master/LICENSE) 
+[![Travis (.com)](https://img.shields.io/travis/com/toparvion/analog/jdk14?style=plastic)](https://travis-ci.com/github/Toparvion/analog) [![Sonar Coverage](https://img.shields.io/sonar/coverage/Toparvion_analog?server=https%3A%2F%2Fsonarcloud.io&style=plastic)](https://sonarcloud.io/dashboard?id=Toparvion_analog) [![Sonar Quality Gate](https://img.shields.io/sonar/quality_gate/Toparvion_analog?server=https%3A%2F%2Fsonarcloud.io&style=plastic)](https://sonarcloud.io/dashboard?id=Toparvion_analog) [![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/toparvion/analog?color=green&sort=semver&style=plastic)](https://github.com/Toparvion/analog/releases/latest) [![GitHub](https://img.shields.io/github/license/toparvion/analog?style=plastic)](https://github.com/Toparvion/analog/blob/master/LICENSE) 
```

Comment:
This is changed just to reflect the current state of the build in the branch (mostly for learning purposes). Therefore it should be reverted before the merge:
```suggestion
[![Travis (.com)](https://img.shields.io/travis/com/toparvion/analog?style=plastic)](https://travis-ci.com/github/Toparvion/analog) [![Sonar Coverage](https://img.shields.io/sonar/coverage/Toparvion_analog?server=https%3A%2F%2Fsonarcloud.io&style=plastic)](https://sonarcloud.io/dashboard?id=Toparvion_analog) [![Sonar Quality Gate](https://img.shields.io/sonar/quality_gate/Toparvion_analog?server=https%3A%2F%2Fsonarcloud.io&style=plastic)](https://sonarcloud.io/dashboard?id=Toparvion_analog) [![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/toparvion/analog?color=green&sort=semver&style=plastic)](https://github.com/Toparvion/analog/releases/latest) [![GitHub](https://img.shields.io/github/license/toparvion/analog?style=plastic)](https://github.com/Toparvion/analog/blob/master/LICENSE) 
```